linera-views/src/reentrant_collection_view.rs

```diff
@@ -960,6 +1196,80 @@ where
     }
 }
 
+impl<C, I, W> ReentrantCustomCollectionView<C, I, W>
+where
+    C: Context + Send + Clone + 'static,
+    ViewError: From<C::Error>,
+    I: Sync + Clone + Send + Debug + Serialize + DeserializeOwned,
+    W: View<C> + Send + Sync + 'static,
+{
+    /// Load multiple entries for write at once.
+    /// The entries in vector_indices have to be all distincts.
+    /// ```rust
+    /// # tokio_test::block_on(async {
+    /// # use linera_views::memory::{create_test_context, MemoryContext};
+    /// # use linera_views::reentrant_collection_view::ReentrantCustomCollectionView;
+    /// # use linera_views::register_view::RegisterView;
+    /// # use crate::linera_views::views::View;
+    /// # let context = create_test_context();
+    ///   let mut view : ReentrantCustomCollectionView<_, u128, RegisterView<_,String>> = ReentrantCustomCollectionView::load(context).await.unwrap();
+    ///   let vector_indices = vec![23,42];
```

Comment:
no need for `vector_` (here and in most places)
In English, it would have to `vector_of_indices` but `indices` is better.